packages/dropdowns/src/elements/Multiselect/Multiselect.spec.tsx

```diff
@@ -162,6 +162,31 @@ describe('Multiselect', () => {
     expect(getByTestId('multiselect')).toHaveAttribute('data-test-is-hovered', 'true');
   });
 
+  it('renders start icon if provided', () => {
+    const { getByTestId } = render(
+      <ExampleWrapper>
+        <Label data-test-id="label">Label</Label>
+        <Multiselect
+          data-test-id="multiselect"
+          start={<svg data-test-id="icon" />}
+          renderItem={({ value, removeValue }) => (
+            <div data-test-id="tag">
+              {value}
+              <button data-test-id="remove" onClick={() => removeValue()} tabIndex={-1}>
```

Comment:
I noticed some of the `data-test` attributes in this render aren't used such as `data-test-id="label"`, `data-test-id="tag"`, `data-test-id="remove"`, and `data-test-id="multiselect"`. Could these attributes be safely removed?